docs: Remove Grafana from Observability section (#10454)
hashicorp/consul: website/content/docs/k8s/connect/observability/metrics.mdx

    baseURL: http://prometheus-server
```

## Deploying Prometheus (_for demo and non-production use-cases only_)

> The Helm chart contains demo manifests for Prometheus for viewing metrics within the Consul UI as was as through other external observability tools such as Grafana. It can be installed with Helm via `prometheus.enabled`. This manifest is based on the community manifest for Prometheus.

Reviewer: ```suggestion
The Helm chart contains demo manifests for deploying Prometheus. It can be installed with Helm via `prometheus.enabled`. This manifest is based on the community manifest for Prometheus.
```
I removed the sentence about viewing metrics in the UI since it seemed extraneous as this is mentioned in the previous section.

> The Prometheus deployment included with Consu Helm is designed to allow quick bootstrapping for trial and demo use cases and not for production use-cases.

Reviewer: ```suggestion
The Prometheus deployment is designed to allow quick bootstrapping for trial and demo use cases, and is not recommended for production use-cases.
```

> Prometheus will be installed in the same namespace that Consul will be installed in and will be installed

Reviewer: ```suggestion
Prometheus will be installed in the same namespace as Consul, and will be installed
```

> and uninstalled along with the Consul installation. In addition, Grafana can be utilized with Prometheus and separately installed and configured option through its own installation via Helm (an example is provided in the following Learn Guide, [Layer 7 Observability with Prometheus, Grafana, and Kubernetes](https://learn.hashicorp.com/tutorials/consul/kubernetes-layer7-observability?in=consul/kubernetes)).

Reviewer: ```suggestion
and uninstalled along with the Consul installation.

Grafana can optionally be utilized with Prometheus to display metrics. The installation and configuration of Grafana must be managed separately from the Consul Helm chart. The [Layer 7 Observability with Prometheus, Grafana, and Kubernetes](https://learn.hashicorp.com/tutorials/consul/kubernetes-layer7-observability?in=consul/kubernetes)) tutorial provides an installation walkthrough using Helm.
```
